src/distcc.h

```diff
@@ -253,7 +254,7 @@ int dcc_set_action_opt(char **, const char *);
 int dcc_set_output(char **, char *);
 int dcc_set_input(char **, char *);
 int dcc_scan_args(char *argv[], /*@out@*/ /*@relnull@*/ char **orig_o,
-                  char **orig_i, char ***ret_newargv);
+                  char **orig_i, char ***ret_newargv,int * hasgcov);
```

Comment:
Please add a space after the comma in lists.

Reply:
This lines has been resolved.